Documentation/diagnostics/source_link.md

```diff
@@ -0,0 +1,103 @@
+# Source Link
+[Portable PDB](https://github.com/dotnet/core/blob/master/Documentation/diagnostics/portable_pdb.md) introduces a new developer productivity feature called Source Link. Source Link allows a Portable PDB to embed unique information about the assembly's original source code during compilation. The [Roslyn](https://github.com/dotnet/roslyn/wiki/Roslyn%20Overview) compiler flag for Source Link is `/sourcelink:<file>`.
+
+## Why Source Link?
+Most debugging is done against locally built source code on a developer's machine. In this scenario, matching the binary to the source is not difficult. However, there are many debugging scenarios where the original source code is not immediately availble. Two good examples of this are debugging crash dumps or third party libraries. In these scenarios, it can be very difficult for a developer to acquire the exact source code that was built to produce the binary they are debugging. Source Link solves this problem by embedding unique information about the source code (such as a git commit hash) in the Portable PDB. Diagnostic tools, such as debuggers, can use this unique information to retrieve the original source code from hosted services (such as [Github](https://github.com)).
+
+## MSBuild Task
+TBD
```

Comment:
No, both `SourceLink.Create.CommandLine` & `SourceLink.Create.GitHub` work on both dotnet and framework. See the 2.1 test plan for several tests https://github.com/ctaggart/SourceLink/issues/188. The latter makes use of the `dotnet sourcelink-git` tool, and has more features. The former just uses command line tools and is easy to get started with.